tox.ini

```diff
@@ -59,8 +65,11 @@ commands_pre =
     pip install -r requirements-min.txt
 
 [testenv:pyargs]
-changedir = {homedir}
+# changedir = {homedir}
 commands =
+    # FIXME: Not sure why need this for tox to see the C-ext
+    python setup.py build_ext --inplace
+    # End of FIXME
```

Comment:
Commenting out the `changedir` directive will break the purpose of doing `pytest --pyargs`, which is to test the package on the installed version of the package in `site-packages`.  `pip install .` should be installing the build C lib in `site-packages` here.